specs/builder.go

```diff
@@ -13,20 +13,21 @@ import (
 	"github.com/moov-io/iso8583/field"
 	"github.com/moov-io/iso8583/padding"
 	"github.com/moov-io/iso8583/prefix"
+	moovsort "github.com/moov-io/iso8583/sort"
 )
 
-type fieldConstructorFunc func(spec *field.Spec) field.Field
+type FieldConstructorFunc func(spec *field.Spec) field.Field
```

Comment:
Why do you think we need it (and following) to be public?

Reply:
It allows library users to use their own custom fields, encodings, sort methods etc and then register them here before exporting their spec. Currently if you use a custom type the builder will complain.

Reply:
That's fair.